osbuild-composer.spec

```diff
@@ -402,6 +402,7 @@ Requires:   dnf-plugins-core
 Requires:   skopeo
 Requires:   make
 Requires:   python3-pip
+Requires:   fdo-admin-cli
```

Comment:
There's no this package on Fedora yet. That might caused osbuild-composer-test installation error on Fedora.
`nothing provides fdo-admin-cli needed by osbuild-composer-tests-58-1.20220719gitba5e4ab.fc35.x86_64`
This package should not be added here, but install this package in ostree-simplified-installer.sh. And more important is Fedora should keep use fdo container, and update to fdo package on RHEL and CentOS Stream only.

Reply:
removed, thanks.